spec/features/javascript/block_controls_spec.rb

```diff
@@ -27,12 +27,14 @@
 
     within('.spotlight-block-controls') do
       expect(page).to have_css('.st-controls-group', count: 2)
+
       within(first('.st-controls-group')) do
-        expect(page).to have_content 'Standard widgets'
+        expect(page).to have_content 'Exhibit item widgets'
         expect(page).to have_css('.st-block-controls__button')
       end
+
       within(all('.st-controls-group').last) do
-        expect(page).to have_content 'Exhibit item widgets'
+        expect(page).to have_content 'Standard widgets'
```

Comment:
Hmm, this is a bit odd that these sections were reversed (if I'm reading this change correctly).

Any idea what's going on there?  I think this will be an issue for us if these widget sections were swapped.

Reply:
should be fixed now; i guess it had something to do with how block sections are defined now